pycode/memilio-generation/memilio/tools/config.json

```
@@ -0,0 +1,41 @@
+[
+    {
+        "source_file": "/cpp/models/ode_seir/model.cpp",
+        "path_database": "/build",
+        "namespace": "mio::oseir::",
+        "python_module_name": "test_oseir",
+        "optional": {
+            "libclang_library_path": "",
+            "simulation_name": "",
+            "age_group": false,
+            "parameterset_wrapper": false,
+            "target_folder": "/pycode/memilio-generation/memilio/tools"
+        }
+    },
+    {
+        "source_file": "/cpp/models/secir/secir.cpp",
+        "path_database": "/build",
+        "namespace": "mio::",
+        "python_module_name": "test_secir",
+        "optional": {
+            "libclang_library_path": "/tools/modulesystem/spack-22.1/opt/spack/linux-ubuntu20.04-haswell/gcc-9.4.0/llvm-13.0.0-e4gcooqwzwwiiprr7u7nnorvgrqqhl63/lib/libclang.so",
+            "simulation_class": "SecirSimulation",
+            "age_group": true,
+            "parameterset_wrapper": true,
+            "target_folder": "/pycode/memilio-generation/memilio/tools"
```

```suggestion
        "source_file": "/cpp/models/ode_secir/model.cpp",
        "path_database": "/build",
        "namespace": "mio::osecir",
        "python_module_name": "test_secir",
        "optional": {
            "libclang_library_path": "/tools/modulesystem/spack-22.1/opt/spack/linux-ubuntu20.04-haswell/gcc-9.4.0/llvm-13.0.0-e4gcooqwzwwiiprr7u7nnorvgrqqhl63/lib/libclang.so",
            "simulation_class": "Simulation",
            "age_group": true,
            "parameterset_wrapper": true,
            "target_folder": "/pycode/memilio-generation/memilio/tools"
```
Please test locally if changed.